<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcTcpServerWindowApp</class>
 <widget class="QMainWindow" name="AcTcpServerWindowApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="4" colspan="2">
     <widget class="QTextEdit" name="textEdit_Info"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP : </string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_IP</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit_IP">
      <property name="text">
       <string>192.168.17.15</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>port :</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_NetPort</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_NetPort">
      <property name="text">
       <string>19999</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton_ConnectServer">
      <property name="text">
       <string>Connect Server</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_StartServer">
      <property name="text">
       <string>Start Server</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEdit_sendmsg"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButton_Send">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
